app/src/Components/AdminPage/VisitorsTable/VisitorsTable.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import './visitorsTable.css';
-const columnHeader = ['Name', 'Company', 'Time Signed In'];
+const columnHeader = ['Name', 'Company', 'Time In', 'Sign Out'];
```

Comment:
Do we remove Sign Out from header column to be consistent with modal or otherway around?